modules/python/python-helpers.c

```diff
@@ -376,6 +376,18 @@ _py_perform_imports(GList *imports)
   g_list_foreach(imports, _foreach_import, NULL);
 }
 
+const gchar *
+_py_object_repr(PyObject *s, gchar *buf, gsize buflen)
+{
+  const gchar *res;
+
+  PyObject *r = PyObject_Repr(s);
+  if (py_bytes_or_string_to_string(r, &res))
```

Comment:
`PyObject_Repr()` might fail. Shouldn't we add a null-check here? `py_bytes_or_string_to_string()` is not null-safe (`PyBytes_CheckExact`, `PyUnicode_Check`, etc.)

Reply:
nice catch. thanks. fixed this up in the next iteration.